README.md

```diff
@@ -159,6 +159,18 @@ jobs:
 
 The Elixir Problem Matchers in this repository are adapted from [here](https://github.com/fr1zle/vscode-elixir/blob/45eddb589acd7ac98e0c7305d1c2b24668ca709a/package.json#L70-L118). See [MATCHER_NOTICE](MATCHER_NOTICE.md) for license details.
 
+## Action versioning
```

Comment:
Let me know about this: this is my attempt at making sure consumers don't rely on potentially breaking versions (this schema is almost like the "ranges" we implement) 😄